packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -290,83 +312,101 @@ export default function TranslationView() {
 
           const isHebrew = bookId < 40;
           return (
-            <ol
-              className={`flex flex-wrap ${
-                isHebrew ? 'ltr:flex-row-reverse' : 'rtl:flex-row-reverse'
-              }`}
-            >
-              {verse.words.map((word, i) => {
-                const targetGloss = targetGlosses[i];
-                const isSaving = glossRequests.some(
-                  ({ wordId }) => wordId === word.id
-                );
+            <>
+              <p
```

Comment:
Once #231 merges, we'll want to set the direction from the language on this element so it matches regardless of the interface language

Reply:
#231 is ready to merge now 😄

Reply:
Done. Tested with arabic